tldr/dfg_extractor.py

```
@@ -3738,3 +3745,168 @@ def extract_luau_dfg(code: str, function_name: str) -> DFGInfo:
+                name = self.get_node_text(left)
+                if self._is_valid_var_name(name):
+                    if is_augmented:
+                        self._add_ref(name, "use", left)
+                        self._add_ref(name, "update", left)
+                    else:
+                        self._add_ref(name, "definition", left)
+            else:
+                self._visit_node(left)
+
+
+def _find_zig_function_by_name(root: Any, name: str, source: bytes) -> Optional[Any]:
```

Ruff (0.14.14)

Dynamically typed expressions (typing.Any) are disallowed in `root`

(ANN401)